spec/system/admin/users_system_spec.rb

```diff
@@ -167,4 +167,24 @@
       end
     end
   end
+
+  describe "Resourse Dropdown List: Validate Order" do
```

Comment:
Can we change this to a request spec? System specs are slow and flaky and should only be used if there needs to be browser / JavaScript interaction.